docs/src/4.15.GLText.mdx

```diff
@@ -7,3 +7,9 @@ import { GLText } from "./jsx/allLiveEditors";
 The prop types are identical to `<Text />`, except that additionally each Text object supports an optional `billboard?: boolean` property. If true (the default), text will always face the camera; if false, text may be rotated in 3D according to `pose.orientation`.
 
 <GLText />
+
+## Scale Invariance
+
+`<GLText />` also supports rendering text at a constant scale regardless of the zoom level by using the optional `scaleInvariantFontSize?: number` property. If set (default is `undefined`), the text will be rendered always at the same size, resembling the behavior of the `<Text />` command. 
```

Comment:
Sorry if this has been asked before; I haven't read through all the prior comments. Did you consider reusing the `scale.z` for this (as Text does; http://wiki.ros.org/rviz/DisplayTypes/Marker#View-Oriented_Text_.28TEXT_VIEW_FACING.3D9.29_.5B1.1.2B-.5D), rather than using a separate prop for `scaleInvariantFontSize`?

Reply:
Hmm, additional thought — it seems like this design doesn't allow different text markers to be different sizes (while all being scale-invariant), is that correct?

Reply:
@jtbandes it does support different sizes based on the `scale` property for markers, but the size remains constant regardless of the zoom level.

Reply:
Oh I see, so the scaleInvariantFontSize is multiplied by the scale? That makes sense, although I think the documentation isn't clear about this, maybe you can update to clarify more

Reply:
I think this would be greatly clarified by an example, e.g. in addition to the example above. E.g. using a couple of different text markers with different sizes, and some with `billboard: true` and some with `billboard: false`.

Reply:
I included a live example GLText's doc that shows how to use scaleInvariantFontSize. It's not as pretty as the matrix one, but it shows how scale invariance behaves compared to a regular GLText object. I also included a Grid that hopefully helps visualize this behavior.